portfolio/src/main/webapp/js/script.js

```diff
@@ -31,12 +31,17 @@ function addRandomFact(id) {
   factContainer.innerHTML = fact;
 }
 
-
-function validate() {
-    var phone = $("#telephone").val();
-    if (phone.length != 10 || isNaN(phone)) {
-        alert("Phone  number is not valid or not valid format.");
-        return false;
-    }
+function validatePhoneNumber() {
+  var phone = $("#telephone").val();
+  var phoneNum = /^[+]?(1\-|1\s|1|\d{3}\-|\d{3}\s|)?((\(\d{3}\))|\d{3})(\-|\s)?(\d{3})(\-|\s)?(\d{4})$/g;
+  if(phone.match(phoneNum))
+  {
+    document.getElementById('invalid').innerHTML = "<p style='color:gray'> Phone Number</p>";
+	return true;
+  }
```

Comment:
I think the following format is more common.

if (condition) {
  ...
} else {
  ...
}